diff --git a/node_modules/@ledgerhq/hw-app-eth/.DS_Store b/node_modules/@ledgerhq/hw-app-eth/.DS_Store
new file mode 100644
index 0000000..6538df2
Binary files /dev/null and b/node_modules/@ledgerhq/hw-app-eth/.DS_Store differ
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.d.ts
index b1044d5..6ff52af 100644
--- a/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.d.ts
+++ b/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.d.ts
@@ -14,7 +14,7 @@ export type StarkQuantizationType = "eth" | "erc20" | "erc721" | "erc20mintable"
  * import Eth from "@ledgerhq/hw-app-eth";
  * const eth = new Eth(transport)
  */
-export default class Eth {
+declare class Eth {
     transport: Transport;
     loadConfig: LoadConfig;
     setLoadConfig(loadConfig: LoadConfig): void;
@@ -353,4 +353,6 @@ export default class Eth {
      */
     provideDomainName(data: string): Promise<boolean>;
 }
+export { Eth };
+export default Eth;
 //# sourceMappingURL=Eth.d.ts.map
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.js b/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.js
index 05972b0..64c9a6f 100644
--- a/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.js
+++ b/node_modules/@ledgerhq/hw-app-eth/lib-es/Eth.js
@@ -38,7 +38,7 @@ const remapTransactionRelatedErrors = e => {
  * import Eth from "@ledgerhq/hw-app-eth";
  * const eth = new Eth(transport)
  */
-export default class Eth {
+class Eth {
     setLoadConfig(loadConfig) {
         this.loadConfig = loadConfig;
     }
@@ -959,4 +959,6 @@ export default class Eth {
         });
     }
 }
+export { Eth };
+export default Eth;
 //# sourceMappingURL=Eth.js.map
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/Eth.d.ts b/node_modules/@ledgerhq/hw-app-eth/lib/Eth.d.ts
index b1044d5..6ff52af 100644
--- a/node_modules/@ledgerhq/hw-app-eth/lib/Eth.d.ts
+++ b/node_modules/@ledgerhq/hw-app-eth/lib/Eth.d.ts
@@ -14,7 +14,7 @@ export type StarkQuantizationType = "eth" | "erc20" | "erc721" | "erc20mintable"
  * import Eth from "@ledgerhq/hw-app-eth";
  * const eth = new Eth(transport)
  */
-export default class Eth {
+declare class Eth {
     transport: Transport;
     loadConfig: LoadConfig;
     setLoadConfig(loadConfig: LoadConfig): void;
@@ -353,4 +353,6 @@ export default class Eth {
      */
     provideDomainName(data: string): Promise<boolean>;
 }
+export { Eth };
+export default Eth;
 //# sourceMappingURL=Eth.d.ts.map
diff --git a/node_modules/@ledgerhq/hw-app-eth/lib/Eth.js b/node_modules/@ledgerhq/hw-app-eth/lib/Eth.js
index 7fd2adf..6b2b5f1 100644
--- a/node_modules/@ledgerhq/hw-app-eth/lib/Eth.js
+++ b/node_modules/@ledgerhq/hw-app-eth/lib/Eth.js
@@ -979,5 +979,6 @@ class Eth {
         });
     }
 }
+exports.Eth = Eth;
 exports.default = Eth;
 //# sourceMappingURL=Eth.js.map
diff --git a/node_modules/@ledgerhq/hw-app-eth/src/Eth.ts b/node_modules/@ledgerhq/hw-app-eth/src/Eth.ts
index 42c1e7a..7123b39 100644
--- a/node_modules/@ledgerhq/hw-app-eth/src/Eth.ts
+++ b/node_modules/@ledgerhq/hw-app-eth/src/Eth.ts
@@ -50,7 +50,7 @@ const remapTransactionRelatedErrors = e => {
  * const eth = new Eth(transport)
  */
 
-export default class Eth {
+class Eth {
   transport: Transport;
   loadConfig: LoadConfig;
 
@@ -1265,3 +1265,5 @@ export default class Eth {
     return true;
   }
 }
+export { Eth };
+export default Eth;
